packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js

```diff
@@ -1,10 +1,12 @@
 export const BANDWIDTH_OUT = 'bandwidth_instance_out.consumption';
 export const FILTER_PRIVATE_NETWORK_BAREMETAL = 'ovh.baremetal';
-
+export const MODE_PRIVATE = 'private_mode';
+export const MODE_PUBLIC = 'public_mode';
+export const MODE_LOCAL_PRIVATE = 'local_private_mode';
```

Comment:
your constants are in french spelling

```suggestion
export const PRIVATE_MODE = 'private_mode';
export const PUBLIC_MODE = 'public_mode';
export const LOCAL_PRIVATE_MODE = 'local_private_mode';
```

Reply:
constants refactored to more english style done